app/services/LocationService.js

```diff
@@ -256,7 +256,7 @@ export default class LocationServices {
           () =>
             Alert.alert(
               languages.t('label.ACCESS2'),
-              'Would you like to open location settings?',
+              languages.t('label.ACCESS3'),
```

Comment:
there are 2 more occurrences of this string in this file I think, maybe replace those too?

Reply:
yes! I fixed them too.